tools/cldr-code/src/main/java/org/unicode/cldr/util/Zipper.java

```diff
@@ -0,0 +1,66 @@
+package org.unicode.cldr.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+public class Zipper {
```

Comment:
It's strange that java.util.zip includes all the low-level functions to implement this, but doesn't include a high-level function or class we could use instead of making our own.